reseq_ros2/reseq_ros2/agevar.py

```diff
@@ -98,21 +95,6 @@ def kinematic(self, linear_vel, angular_vel, yaw_angle):
         
         return linear_out, angular_out
 
-    # compute motors velocity for each module
-    def vel_motors(self, lin_vel, ang_vel, sign):
```

Comment:
I'm not too sure I understand how motor velocities (for the real robot, not just in gazebo) will be computed without this function

Reply:
See https://github.com/Team-Isaac-Polito/reseq_ros2/pull/5#discussion_r1217712757